### ISSUE TITLE
Remove 'include = cat /usr/share/mr/vcsh' from .mrconfig

### ISSUE THREAD

**Issue by martinburger** [RichiH/vcsh_mr_template#2]
In `.mrconfig`, file `/usr/share/mr/vcsh` gets included.

Because vcsh support has been moved into mr, this is not required anymore.

Moreover, that include breaks mr on OS X when using Homebrew because the included file resides at `/usr/local/share/mr/vcsh` instead:

```
cat: /usr/share/mr/vcsh: No such file or directory
mr: include command exited nonzero (256)
```

**Comment by RichiH** [comment]
Thanks; fixed in c213fce87feed774ae8ce7aa5696cbd92b9da5be

Everything else working for you?

**Comment by martinburger** [comment]
I think the combo vcsh / myrepos is brilliant. It facilitates managing dotfiles across different machines tremendously.

As chance would have it :wink:, I just published a blog entry that explains how to set up these tools in order that you are able to [manage dotfiles quickly and effortlessly](http://www.martin-burger.net/blog/unix-shell/manage-dotfiles-quickly-and-effortlessly/).

Perhaps you would like to add a link to my article in vcsh's [README file](https://github.com/RichiH/vcsh/blob/master/README.md)?
